From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Thu, 5 Mar 2020 09:45:17 +0000
Subject: [PATCH] Export GNULIB_TOOL for libtextstyle autogen.sh

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 0aa68224ea01a586a2e43c2b9b6a30a65a8c6449..1cbf660d23672020a87ea0a4fe8290967b9216f2 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -660,7 +660,7 @@ cd "$dir0"
 echo "$0: generating files in libtextstyle..."
 cd libtextstyle
 (if ! $skip_gnulib; then export GNULIB_SRCDIR; fi
- ./autogen.sh $skip_gnulib_option
+ GNULIB_TOOL=$GNULIB_TOOL ./autogen.sh $skip_gnulib_option
 ) || exit $?
 cd "$dir0"
 
