From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Thu, 5 Mar 2020 09:45:17 +0000
Subject: [PATCH] Export GNULIB_TOOL for libtextstyle autogen.sh

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index b896c3f1e3a362d7f60f9eb4888543a5226b22ec..2568a2415f23d5fa44e16c8427ff7cdad9e0abb6 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -449,7 +449,7 @@ cd "$dir0"
 echo "$0: generating files in libtextstyle..."
 cd libtextstyle
 (if ! $skip_gnulib; then export GNULIB_SRCDIR; fi
- ./autogen.sh $skip_gnulib_option
+ GNULIB_TOOL=$GNULIB_TOOL ./autogen.sh $skip_gnulib_option
 ) || exit $?
 cd "$dir0"
 
