From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Thu, 5 Mar 2020 09:46:38 +0000
Subject: [PATCH] Disable man and doc from gettext-runtime

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 gettext-runtime/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gettext-runtime/Makefile.am b/gettext-runtime/Makefile.am
index 18946894161f4d86f19cba12072a50ce6ae1fcac..6586dda67f34bc94d046fae4066fd5460a328b3c 100644
--- a/gettext-runtime/Makefile.am
+++ b/gettext-runtime/Makefile.am
@@ -26,7 +26,7 @@ SUBDIR_libasprintf = libasprintf
 else
 SUBDIR_libasprintf =
 endif
-SUBDIRS = doc intl intl-java intl-csharp intl-d intl-modula2 gnulib-lib $(SUBDIR_libasprintf) src po man m4 tests install-tests
+SUBDIRS = intl intl-java intl-csharp intl-d intl-modula2 gnulib-lib $(SUBDIR_libasprintf) src po m4 tests install-tests
 
 EXTRA_DIST = BUGS
 
