From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Thu, 5 Mar 2020 09:48:52 +0000
Subject: [PATCH] Disable doc from libtextstyle

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 libtextstyle/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libtextstyle/Makefile.am b/libtextstyle/Makefile.am
index 27d631b2f900ca09a9210eb8bade15bc702a0680..0f788f2068f1cf9b501ddbb7e94e36764f19d727 100644
--- a/libtextstyle/Makefile.am
+++ b/libtextstyle/Makefile.am
@@ -20,7 +20,7 @@ AUTOMAKE_OPTIONS = 1.13 gnu no-dependencies
 ACLOCAL_AMFLAGS = -I m4 -I gnulib-m4
 
 # The list of subdirectories containing Makefiles.
-SUBDIRS = lib tests adhoc-tests doc
+SUBDIRS = lib tests adhoc-tests
 
 EXTRA_DIST = \
   version.sh \
